tcms_tenants/templates/tcms_tenants/new.html

```diff
@@ -68,6 +62,13 @@
             {% block extra_contents %}
                 {{ form.paid_until }}
             {% endblock %}
+
+            <div class="form-group">
+                <label class="col-md-1 col-lg-1" for="id_logo">{% trans "Logo" %}</label>
+                {% if object %}
+                    {% include 'tcms/templates/include/attachments.html' %}
+                {% endif %}
```

Comment:
As-is the attachment form will occupy the entire screen. Surround with a `div` element and specify the same column sizes as the rest of this page. See how it is done in `tcms/testplans/templates/testplans/get.html`.